src/EnergyPlus/UnitarySystem.cc

```diff
@@ -991,6 +991,15 @@ namespace UnitarySystems {
             DataLoopNode::Node(this->m_HeatRecoveryInletNodeNum).MassFlowRate = mdotHR;
         }
 
+        if (this->m_FanOpModeSchedPtr > 0) {
```

Comment:
Well, I actually suggested this be moved to line 1006, or better yet 1010, below since it needs only be done once per time step. Might as well make that change while you correct the unit test.